matsim/src/main/java/org/matsim/core/router/NetworkRoutingInclAccessEgressModule.java

```diff
@@ -263,7 +263,7 @@ private static Activity createInteractionActivity( final Coord interactionCoord,
 	}
 
 
-	private static double routeBushwhackingLeg(Person person, Leg leg, Coord fromCoord, Coord toCoord, double depTime,
+	static double routeBushwhackingLeg(Person person, Leg leg, Coord fromCoord, Coord toCoord, double depTime,
```

Comment:
Not sure, would it probably make sense to move this to a separate `TeleportationUtils` class or similar? Because that's what it calculates in the end.

At least I find it confusing that the `FallbackRoutingModuleDefaultImpl` uses functionality from the `NetworkRoutingInclAccessEgressModule`, when all it actually does is to calculate a hard-coded teleportation.

Also, this method has hard-coded assumptions about teleportation speed, with a `yyyyyy`-remark to take the values from the config. For the fallback it might not make a difference, although one could argue that the fallback is "teleported walking" and that it should try to use the walk-parameters, if `walk` is defined for teleportation.

Reply:
Agreed.  I just didn't want to move it before we agree in general.  Do we all like `TeleportationUtils`?